skins/cat17/address-form/src/AddressForm.vue

```diff
@@ -122,29 +114,33 @@
 				}
 			}
 		},
+		computed: {
+            fieldErrors(): AddressValidity {
+                return {
+                    salutation: this.$store.state.form.salutation.isValid === Validity.INVALID,
+                    companyName: this.$store.state.form.companyName.isValid === Validity.INVALID,
+                    firstName: this.$store.state.form.firstName.isValid === Validity.INVALID,
+                    lastName: this.$store.state.form.lastName.isValid === Validity.INVALID,
+                    street: this.$store.state.form.street.isValid === Validity.INVALID,
+                    city:  this.$store.state.form.city.isValid === Validity.INVALID,
+                    postcode: this.$store.state.form.postcode.isValid === Validity.INVALID
+                }
```

Comment:
Just a suggestion:
```
return Object.keys(formData).reduce((acc, field) => {
    if (!formData[field].optionalField) {
        acc[field] = this.$store.state.form[field].isValid === Validity.INVALID
    }
    return acc;
}, {});
```
It's less clear what's going on but on the other hand it's less repetitive. Also if we have to change something about the current set of fields we have to do it only in one place.